pkg/config/errors.go

```diff
@@ -43,6 +60,15 @@ func (err *InvalidSettingError) Error() string {
 	return fmt.Sprintf("invalid setting provided to a module, cause: %v", err.cause)
 }
 
+// InvalidModuleError signifies a problem with the supplied module name.
+type InvalidModuleError struct {
+	modID ModuleID
```

Comment:
This field needs to be exported. Start with capital letter.
Let's keep it short
```go
type InvalidModuleError struct {
  ID ModuleID
}
```